src/main/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingService.kt

```diff
@@ -54,5 +62,42 @@ class ArbeidsfordelingService(
         }
     }
 
+    fun finnBehandlendeEnhetForPersonMedRelasjoner(personIdent: String, tema: Tema): List<Enhet> {
+        return cacheManager.getValue("navEnhet", personIdent) {
+            val personMedRelasjoner = personopplysningerService.hentPersonMedRelasjoner(personIdent, tema)
+
+            val aktuellePersoner: List<PersonMedAdresseBeskyttelse> =
+                    listOf(PersonMedAdresseBeskyttelse(personMedRelasjoner.personIdent, personMedRelasjoner.adressebeskyttelse)) +
+                    personMedRelasjoner.barn +
+                    personMedRelasjoner.barnsForeldrer +
+                    personMedRelasjoner.sivilstand
+
+            val egneAnsatte = finnEgneAnsatte(aktuellePersoner)
+            val identMedStrengeste = utledPersonIdentMedStrengestSjekk(personIdent = personIdent,
+                                                                       personerMedAdresseBeskyttelse = aktuellePersoner,
+                                                                       egneAnsatte = egneAnsatte)
+            val personInfo = personopplysningerService.hentPersoninfo(identMedStrengeste)
```

Comment:
Kan man bruke `hentGeografiskTilknytning` fra PDL her?

Reply:
Ja, kanskje vi skal prøve det! Og forsåvidt diskresjonskode fra PDL og slutte å bruke TPS for den nye varianten?